arcgis-ios-sdk-samples/Route & Directions/Find route/README.metadata.json

```diff
@@ -0,0 +1,39 @@
+{
+    "category": "Route and Directions",
```

Comment:
Does this need to be Route & Navigation to match the folder structure in the sample repo? ... or is it OK to be Route and Navigation (which is more readable).

Reply:
It should be fine: what matters the most is that `category` property values share identical strings if they have the same category.